blueprints/ember-cli-addon-docs/index.js

```diff
@@ -63,9 +63,39 @@ module.exports = {
     }
 
     if (!hasPlugins) {
+      return this._chooseAddonsToInstall()
+        .then((addon) => {
+          return this.addAddonsToProject({
+            packages: [addon],
+            blueprintOptions: {
+              save: options.save
+            }
+          });
+        });
       return this.addAddonsToProject({
         packages: ['ember-cli-addon-docs-yuidoc']
       });
     }
+  },
+
+  _chooseAddonsToInstall() {
+    // Ask which ember addon to install
+    return this.ui.prompt({
+      type: 'list',
+      name: 'addonToInstall',
+      message: 'Which documentation style would you like to use?',
```

Comment:
Should we include a choice not to install anything at all? I could imagine wanting to use another addon-docs plugin that isn't necessarily part of the ember-learn org, or just not needing or wanting generated API docs for an addon at all.

Reply:
Fair enough. I'll probably turn around and add a "no plugin" option anyway because I think it's valuable, but I don't need to push that work on you here 🙂